### PR TITLE
Changed Chunk size to 1 MB, to allow sending large files

### PR DESCRIPTION
**Pull request by nnk03**

### CODE REVIEW

**Review by dshreddy — COMMENTED** [review]
LGTM, let me know once testing is done

File: FileCloner/Models/Constants.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class Constants
         public static string IPAddress = GetIP();
 
         // Size of FileChunk to be sent over network
-        public static int FileChunkSize = 4096;
+        public static int FileChunkSize = 1024 * 1024;
```
Comment (dshreddy) [inline FileCloner/Models/Constants.cs:51]: Noted

File: FileCloner/Models/NetworkService/Client.cs [hunk 2]
```diff
@@ -346,18 +355,39 @@ public void OnFileForCloningReceived(Message data)
                     return;
                 }
 
-                int chunkNumber = int.Parse(messageBodyList[0]);
+                string[] chunkNumberByNumberOfTransmissionsRequired = messageBodyList[0].Split('/', 2);
+                if (chunkNumberByNumberOfTransmissionsRequired.Length != 2) 
+                {
+                    // code should never reach here
+                    logAction?.Invoke("Sending Files in Wrong Format");
```
Comment (dshreddy) [inline FileCloner/Models/NetworkService/Client.cs:362]: Change this to "[Client] Sending Files in Wrong Format"

**Review by dshreddy — COMMENTED** [review]
Let me know after testing is done

**Comment by nnk03** [comment]
Closing this pull request, because of some bugs
Reverted back to a previous commit and created a new branch and resolved the bugs
Made a separate PR  #6

**Pull request closed without merging**